### ISSUE TITLE
Notification to a group does not work as advertised

### ISSUE THREAD

**Issue by mstolt** [home-assistant/companion.home-assistant#1022]
As described on the following page I created a notification group with two devices. That did not work. When sending a Notification I receive an error:

> Die Automatisierung „Experiment Notification“ (automation.experiment_notification) hat eine Aktion, die einen unbekannten Dienst aufruft: notify.matthias_computer.

When sending a notification to each individual device it works fine. So the general configuration it good.

https://companion.home-assistant.io/docs/notifications/notifications-basic/#sending-notifications-to-multiple-devices

These are my configuration bits. 

In configuration.yaml:

```
notify: 
  - platform: group
    name: MATTHIAS_COMPUTER
    services: 
      - service: mobile_app_porto
      - service: mobile_app_neaped
```

in the Automation

```
alias: Experiment Notification
description: ""
trigger:
  - platform: state
    entity_id:
      - input_button.experiment_trigger
condition: []
action:
  - service: notify.MATTHIAS_COMPUTER
    data:
      message: "Test"
mode: single
```

Any advice on this? Is there anything wrong in the documentation?

**Comment by dshokouhi** [comment]
Try using a lower case notify group name so the service call is all lower case

**Comment by mstolt** [comment]
> Try using a lower case notify group name so the service call is all lower case

Thanks, I actually already tried that. But it did noch change anything. It still don't work.

**Comment by dshokouhi** [comment]
are you certain the service call was created? the error suggests it does not exist.

**Comment by mstolt** [comment]
you are right. I changed all configuration files to create the group, I changed the automations, I even reloaded the YAML files, but it did not work. Then I thought it could help to post the English Error Messages, so I switched the language and restarted HomeAssistent. Et voila! It worked. The Notification Groups seems to need a restart of Home Assistant. 

Tomorrow I'll test if the group can contain UPPERCASE (just curious). I'll post an update. 

Thank you.
